File: php/lecture/privateVar/Buku.php
<?php

class Buku{
    private $judul = "";
    private $pengarang = "";

    function __construct($j = "", $p = ""){
        $this->judul = $j;
        $this->pengarang = $p;
    }

    function setJudul($j = ""){
        $this->judul = $j;
    }
    
    function setPengarang($p = ""){
        $this->pengarang = $p;
    }

    function getJudul(){
        return $this->judul;
    }

    function getPengarang(){
        return $this->judul;
    }
}
?>

File: java/lecture/PrivateVar/Buku.java
class Buku{
    private String judul;
    private String pengarang;

    Buku(){
    }

    Buku(String j, String p){
        this.judul = j;
        this.pengarang = p;
    }

    void setJudul(String j){
        this.judul = j;
    }

    void setPengarang(String p){
        this.pengarang = p;
    }

    String getJudul(){
        return this.judul;
    }

    String getPengara(){
        return this.pengarang;
    }
}